Al ejecutar pylint aparecen advertencias

# Error en sitio web de python Barranquilla

## Comportamiento esperado

Al ejecutar pylint no deberian aparecer advertencias

## Comportamiento actual

Aparecen las siguientes advertencias al ejecutar pylint


```
$ pylint --recursive=y tests scripts
************* Module tests.conftest
tests/conftest.py:7:0: C0411: standard import "from typing import Generator" should be placed before "import pytest" (wrong-import-order)
************* Module events
scripts/events.py:6:0: C0301: Line too long (159/100) (line-too-long)
scripts/events.py:7:0: C0301: Line too long (137/100) (line-too-long)
scripts/events.py:15:50: C0303: Trailing whitespace (trailing-whitespace)
scripts/events.py:16:0: C0304: Final newline missing (missing-final-newline)
scripts/events.py:4:1: W0511: TODO: Create logic for oauth authentication when required (fixme)
scripts/events.py:1:0: C0114: Missing module docstring (missing-module-docstring)
scripts/events.py:6:22: W3101: Missing timeout argument for method 'requests.get' can cause your program to hang indefinitely (missing-timeout)
scripts/events.py:7:24: W3101: Missing timeout argument for method 'requests.get' can cause your program to hang indefinitely (missing-timeout)
scripts/events.py:15:5: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)

-------------------------------------------------------------------
Your code has been rated at 7.92/10 (previous run: 10.00/10, -2.08)
```

## Pasos para reproducir el error

1. Ejecutar `pylint --recursive=y tests scripts` en la carpeta raiz del proyecto

--- a/scripts/events.py
+++ b/scripts/events.py
@@ -1,10 +1,16 @@
+"""Descarga eventos usando Meetup api"""
+
 import json
 import requests
 
-# TODO: Create logic for oauth authentication when required
+# NOTE: This endpoint no requires authentication, but this API is deprecated
 
-past_events_request = requests.get('https://api.meetup.com/pythonbaq/events?&sign=true&photo-host=secure&page=200&fields=featured_photo&desc=true&status=past')
-future_events_request = requests.get('https://api.meetup.com/pythonbaq/events?&sign=true&photo-host=secure&page=3&fields=featured_photo')
+EVENTS_URL = 'https://api.meetup.com/pythonbaq/events?&sign=true&photo-host=secure'
+PAST_EVENTS_URL = EVENTS_URL + '&page=200&fields=featured_photo&desc=true&status=past'
+FUTURE_EVENTS_URL = EVENTS_URL + '&page=3&fields=featured_photo'
+
+past_events_request = requests.get(PAST_EVENTS_URL, timeout=30000)
+future_events_request = requests.get(FUTURE_EVENTS_URL, timeout=30000)
 
 
 meetup_responses = json.dumps({
@@ -12,5 +18,5 @@ meetup_responses = json.dumps({
     "future_events": future_events_request.json()
 })
 
-with open("databags/meetup.json", "w") as outfile: 
-    outfile.write(meetup_responses) 
+with open("databags/meetup.json", "w", encoding="utf-8") as outfile:
+    outfile.write(meetup_responses)

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -3,8 +3,8 @@ Configuración de test automatizados
 """
 
 import os
-import pytest
 from typing import Generator
+import pytest
 from selenium.webdriver import Remote
 from selenium.webdriver.chrome.options import Options as ChromeOptions
 from selenium.webdriver.firefox.options import Options as FirefoxOptions

